Formula/container-diff.rb

```diff
@@ -0,0 +1,254 @@
+      -perl                              5.18.2-2ubuntu1                         16.9M
+      -perl-base                         5.18.2-2ubuntu1                         4.7M
+      -perl-modules                      5.18.2-2ubuntu1                         15.8M
+      -plymouth                          0.8.8-0ubuntu17.1                       457K
+      -procps                            1:3.3.9-1ubuntu2.2                      680K
+      -python3                           3.4.0-0ubuntu2                          100K
+      -python3-minimal                   3.4.0-0ubuntu2                          139K
+      -python3.4                         3.4.0-2ubuntu1                          325K
+      -python3.4-minimal                 3.4.0-2ubuntu1                          4M
+      -readline-common                   6.3-4ubuntu2                            109K
+      -resolvconf                        1.69ubuntu1.1                           245K
+      -rsync                             3.1.0-2ubuntu0.1                        634K
+      -rsyslog                           7.4.4-1ubuntu2.5                        1.5M
+      -sed                               4.2.2-4ubuntu1                          304K
+      -sensible-utils                    0.0.9                                   110K
+      -sudo                              1.8.9p5-1ubuntu1.1                      1.5M
+      -sysv-rc                           2.88dsf-41ubuntu6.1                     221K
+      -sysvinit-utils                    2.88dsf-41ubuntu6.1                     242K
+      -tar                               1.27.1-1                                784K
+      -tzdata                            2015d-0ubuntu0.14.04                    1.5M
+      -ubuntu-keyring                    2012.05.19                              46K
+      -ubuntu-minimal                    1.325                                   60K
+      -ucf                               3.0027 nmu1                             229K
+      -udev                              204-5ubuntu20.11                        5M
+      -upstart                           1.12.1-0ubuntu4.2                       1.6M
+      -ureadahead                        0.100.0-16                              124K
+      -util-linux                        2.20.1-5.1ubuntu20.4                    1.6M
+      -vim-common                        2:7.4.052-1ubuntu3                      317K
+      -vim-tiny                          2:7.4.052-1ubuntu3                      931K
+      -whiptail                          0.52.15-2ubuntu5                        85K
+      -xauth                             1:1.0.7-1ubuntu1                        86K
+      -xkb-data                          2.10.1-1ubuntu1                         2.8M
+      -zlib1g                            1:1.2.8.dfsg-1ubuntu1                   171K
+    EOS
+
+    assert_match expected, shell_output("#{bin}/container-diff analyze docker/whalesay", 0)
```

Comment:
The `, 0` is redundant here. That's already the default check with `shell_output`.